docs/tutorial/curation.rst

```diff
@@ -473,11 +473,130 @@ a network based optimization to identify metabolites with no production pathways
+included gap-filling reactions with their associated penalty values. And lastly will list any reactions where the
+gap-filling result suggests that the flux bounds of the reaction be changed. A sample of the reaction is shown below:
+
+.. code-block:: shell
+
+    ....
+    TPI	Model	0	Dihydroxyacetone-phosphate[c] <=> Glyceraldehyde-3-phosphate[c]
+    rxn1	Model	0	Succinate[c] => a[c]
+    rxn3	Model	0	b[c] => c[c] + d[c]
+    rxn2	Add	1	a[c] => b[c]
+
+Some additional options can be used to refine the gap-filling. The first of these options is ``--no-implicit-sinks``
+option that can be added to the command. If this option is used then the gap-filling will be performed with no
+implicit sinks for compounds, meaning that all compounds produced need to be consumed by other reactions in the
+metabolic model. By default, if this option is not used with the command, then implicit sinks are added for all
+compounds in the model meaning that any compound that is produced in excess can be removed through the added sinks.
+
+The other way to refine the gap-filling procedure is through defining specific penalty values for the addition of
+reactions from different sources. reactions can be set for specific reactions in a gap-filling database
+through a tab separated file provided in the command using the ``--penalty`` option. Additionally penalty values
+for all database reactions can be set using the ``--db-penalty`` option followed by a penalty value. Similarly
+penalty values can be assigned to added transport reactions using the ``--tp-penalty`` option and to added
+exchange reactions using the ``--ex-penalty`` option. An example of a command that applies these penalties
+to a gap-filling simulation would be like follows:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model gapfill --compound d[c] --ex-penalty 100 --tp-penalty 10 --db-penalty 1
+
+The ``gapfill`` function in PSAMM can be used through the model construction process to help identify potential
+new reactions to add to a model and to explore how metabolic gaps effect the capabilities of a metabolic
+network.
+
+FastGapFill
+~~~~~~~~~~~
+
+The ``fastgapfill`` function in `PSAMM` is different gap-filling method that uses the FastGapFill algorithm
+to attempt to generate a gap-filled model that is entirely flux consistent [Thiele14]_. The implementation
+of this algorithm in `PSAMM` can be utilized for unblocking an entire metabolic model or for unblocking
+specific reactions in a network. Often times unblocking all of the reactions in a model at the same time
+will not produce the most meaningful and easy to understand results so only performing this function on a
+subset of reactions is preferable. To do this the ``--subset`` option can be used to provide a file that
+contains a list of reactions to unblock. In this example that list would look like this:
+
+.. code-block:: shell
```

Comment:
This is not a command so should not be a `shell` code block. (just change "...would look like this:" to double colon "...would look like this::" and remove the ".. code-block:: shell" line.)